cogs/ahk/ahk.py

```diff
@@ -428,7 +429,18 @@ async def rlx(self, ctx, *, code):
 		'''Compile and run Relax code through CloudAHK. Example: `rlx define i32 Main() {return 20}`'''
 
 		await self.cloudahk_call(ctx, code, 'rlx')
-
+	
+	@commands.command()
+	async def wta(self,ctx):
+		open_category = self.bot.get_channel(OPEN_CATEGORY_ID)
+		channels = list(open_category.text_channels)
+		channel_mentions = []
+		for i, c in enumerate(channels):
+			if c.id == GET_HELP_CHANNEL_ID:
+				continue
+		channel_mentions.append(c.mention)
```

Comment:
Should be inside of the loop, this will only list the final channel iterated over, including `GET_HELP_CHANNEL_ID`

Reply:
Thank you, fixed in https://github.com/Run1e/AceBot/pull/19.